.travis.yml

```diff
@@ -40,6 +40,7 @@ before_install:
 - export AB2D_HICN_HASH_PEPPER=`cat bfd_sbx_hash_pepper`
 - export AB2D_HICN_HASH_ITER=1000
 - export AB2D_CLAIMS_SKIP_BILLABLE_PERIOD_CHECK=true
+- export AB2D_EARLIEST_DATA_DATE=01/01/1900
```

Comment:
It's 1900 here and 2020 elsewhere is that b/c our tests are relying on earlier dates?

Reply:
Correct. The real one is 2020, but the e2e tests rely on test data from 199-2002.